Ckan_API/Ckan_API.py

```diff
@@ -0,0 +1,37 @@
+from urllib.parse import urljoin
+
+import requests
+
+
+class CkanAPICall:
+
+    def __init__(self, ckan_api_url, ckan_api_key):
+        self.ckan_api_url = ckan_api_url
+        self.ckan_api_key = ckan_api_key
+
+    def get_all_datasets(self):
+        packages = self.package_list()
+
+        datasets = []
+        for package in packages:
+            datasets.append(self.package_show(package))
+        return datasets
+
+    def package_list(self):
+        urlpath = self.create_url('/package_list')
+        response = requests.get(urlpath, headers=
+                                {'Authorization': self.ckan_api_key})
+        json_response = response.json()
```

Comment:
DRY. You call these 3 lines in all your methods. How about:

```python
def _send_get_request(self, path, params=None):
        urlpath = self.create_url(path)
        response = requests.get(urlpath, params=params,
                                headers={'Authorization': self.ckan_api_key})
        return response.json()
```
Then your other methods become basically one-liners around this.